### PR TITLE
threatconnect: various changes from testing and customer feedback 

### PR DESCRIPTION
**Pull request by GShepherdTC**
Enhancement

## Proposed commit message

Remove default SSL certificate, update descriptions, add ability to control attributes, associated groups and indicators. Changes came from internal testing and customer feedback.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] New radio sliders added during setup wizard
- [ ] Removed default SSL cert
- [ ] Documentation changes
- [ ] Removed some fields from being added to ES

## How to test this PR locally

Load integration and review the changes.

## Related issues

- 

## Screenshots

-

### CODE REVIEW

File: packages/ti_threatconnect/data_stream/indicator/agent/stream/cel.yml.hbs [hunk 1]
```diff
@@ -1,25 +1,42 @@
 config_version: 2
+# set interval in between polling
```
Comment (efd6) [inline packages/ti_threatconnect/data_stream/indicator/agent/stream/cel.yml.hbs:2]: If we're going to have these comments, let's consistently make them sentences with initial capital and final period.

File: packages/ti_threatconnect/data_stream/indicator/agent/stream/cel.yml.hbs [hunk 2]
```diff
@@ -1,25 +1,42 @@
...
     ioc_expiration_duration: "{{ioc_expiration_duration}}"
 {{/if}}
+
+# This specifies SSL/TLS configuration. If the ssl section is missing, the host’s CAs are used for HTTPS connections.
 {{#if ssl}}
 resource.ssl: {{ssl}}
 {{/if}}
+
+# Controls if SSL is enabled or not which corresponds to verify SSL or not.
+# resource.ssl.enabled: false
```
Comment (efd6) [inline packages/ti_threatconnect/data_stream/indicator/agent/stream/cel.yml.hbs:30]: Why is this being left here?

(also s/if/whether/)

**Comment by efd6** [comment]
@GShepherdTC Why was this closed?

**Comment by GShepherdTC** [comment]
I opened a new one 9836 after merging in the latest changes.

**Comment by efd6** [comment]
Let's keep working on the same PR in future. Each migration means that all the previous discussion has been lost and so review essentially needs to start again.

**Pull request closed without merging**